notes/v0.2.0.md

```diff
@@ -22,15 +22,13 @@ In this release we have:
     `tf.image`.
   - Improved consistency for the creation of initializer ops.
   - Added support for the `tf.initializer` op creation context.
-  - Exposed part of the `TensorArray` API.
+  - The exposed part of the `TensorArray` API.
   - Exposed `tf.Op.Builder` in the public API.
   - Improvements to the learn API:
     - Refactored `mode` into an implicit argument.
     - Improved the evaluator hook.
     - Removed the layer creation context mechanism, to be refactored
-      later. It was causing some issues due to bad design and unclear
-      semantics. The plan is to implement this, in the near future, as
-      wrapper creation context layers.
+      later. It was causing some issues due to bad design and unclear semantics. The plan is to implement this, in the near future, as wrapper creation context layers.
```

Comment:
The reason I split the lines is so that they are easier to edit on the terminal. I try to keep them to 80 characters max length.